content/guides/server-sent-events/src/server_sent_events/service.clj

```diff
@@ -0,0 +1,180 @@
+;; tag::original-service[]
+(ns server-sent-events.service
+  (:require [io.pedestal.http :as http]
+            [io.pedestal.http.route :as route]
+            [io.pedestal.http.body-params :as body-params]
+            [ring.util.response :as ring-resp]))
+
+(defn about-page
+  [request]
+  (ring-resp/response (format "Clojure %s - served from %s"
+                              (clojure-version)
+                              (route/url-for ::about-page))))
+
+(defn home-page
+  [request]
+  (ring-resp/response "Hello World!"))
+
+;; Defines "/" and "/about" routes with their associated :get handlers.
+;; The interceptors defined after the verb map (e.g., {:get home-page}
+;; apply to / and its children (/about).
+(def common-interceptors [(body-params/body-params) http/html-body])
+
+;; Tabular routes
+(def routes #{["/" :get (conj common-interceptors `home-page)]
```

Comment:
I like to column-align the verbs in these routes. It emphasizes the tabular nature and makes it easy for visual scanning.